Trie/UnitTrieTests/TrieUnitTests.cs

```diff
@@ -0,0 +1,49 @@
+using TrieProj;
+using NUnit.Framework;
+
+namespace UnitTrieTests;
+
+public class UnitTests
+{
+    private readonly Trie _trie = new Trie();
+    
+    [SetUp]
+    public void Setup()
+    {
+        _trie.Add("test");
+    }
+
+    
+    [Test]
```

Comment:
```suggestion
    }

    [Test]
```
Две пустые строчки подряд по стайлгайду не пишутся никогда